index.html

```diff
@@ -15,24 +15,52 @@
     <link rel="stylesheet" href="styles.css">
 </head>
 <body>
+    <header>
+        <nav>
+            <a href="#">Awesome Books</a>
+            <ul class="navLinkCont">
+                <li><button type="button" class="navLink" id="list">List</button></li>
+                <li><button type="button" class="navLink" id="addNew">Add new</button></li>
+                <li><button type="button" class="navLink" id="contact">Contact</button></li>
+            </ul>
+        </nav>
+        <span class="date">Hello</span>
+    </header>
     <main>
-        <h1>All awesome Books</h1>
-        <form>
+        <section id="bookList" class="hidden">
+        <h1 class="bookHeader">All awesome Books</h1>
             <table id="bookContainer">
             <!-- <label id="title"></label><br>
             <label id="author"></label><br>
             <input class="remove" type="button" value="Remove">
             <hr> -->
```

Comment:
- [ ] Please remove the commented code.

Reply:
Commented code has been removed, thank you for bringing this to our attention! 🙏